draft-ietf-ccamp-rfc9093-bis.xml

```diff
@@ -151,251 +151,206 @@ Please replace XXXX with the RFC number assigned to this document and remove thi
+that was defined separately in wson-label-start-end and flexi-grid-label-start-end,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See <xref target="label-range"/> for more details.</t>
+  </dd>
+  <dt>wdm-label-hop:</dt>
+  <dd>
+    <t>A YANG grouping that combines the definition of label hop information
+that was defined separately in wson-label-hop and flexi-grid-label-hop,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See <xref target="label-range"/> for more details.</t>
+  </dd>
+  <dt>wdm-label-range-info:</dt>
+  <dd>
+    <t>A YANG grouping that combines the definition of label range information
+that was defined separately in wson-label-range-info and flexi-grid-label-range-info,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See <xref target="label-range"/> for more details.</t>
+  </dd>
+  <dt>wdm-label-step:</dt>
+  <dd>
+    <t>A YANG grouping that combines the definition of label step information
+defined separately in wson-label-step and flexi-grid-label-step,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See <xref target="label-range"/> for more details.</t>
+  </dd>
+  <dt>transceiver-capabilities:</dt>
+  <dd>
+    <t>A YANG grouping to define the transceiver capabilities (also called
 "modes") needed to determine optical signal compatibility.</t>
-</li></ul>
-
-<t>standard-mode:</t>
-
-<ul empty="true"><li>
-  <t>A YANG grouping for the standard modes defined in <xref target="ITU-T_G.698.2"/>.</t>
-</li></ul>
-
-<t>organizational-mode:</t>
-
-<ul empty="true"><li>
-  <t>A YANG grouping to define transponder operational mode supported by
+  </dd>
+  <dt>standard-mode:</dt>
+  <dd>
+    <t>A YANG grouping for the standard modes defined in <xref target="ITU-T_G.698.2"/>.</t>
+  </dd>
+  <dt>organizational-mode:</dt>
+  <dd>
+    <t>A YANG grouping to define transponder operational mode supported by
 organizations or vendors, as defined in <xref target="I-D.ietf-ccamp-optical-impairment-topology-yang"/>.</t>
-</li></ul>
-
-<t>common-explicit-mode:</t>
-
-<ul empty="true"><li>
-  <t>A YANG grouping to define the list of attributes related to optical
+  </dd>
+  <dt>explicit-mode:</dt>
+  <dd>
+    <t>A YANG grouping to define the list of attributes related to optical
 impairments limits in case of transceiver explicit mode, as defined in <xref target="I-D.ietf-ccamp-optical-impairment-topology-yang"/>.</t>
-</li></ul>
-
-<t>transmitter-tuning-range:</t>
-
-<ul empty="true"><li>
-  <t>A YANG grouping that defines the transmitter tuning range, which
+  </dd>
+  <dt/>
+  <dd>
+    <t>Note that the the actual portion of the spectrum occupied by an OTSi is not explicitly reported within the explicit-mode parameters because it can calculated using the available-baud-rate, the roll-off and the min-carrier-spacing attributes.</t>
```

Comment:
Minor comment: may be better to write ".... it can **be** calculated using the ...."

Reply:
Fixed with 844527b1a0d069cb34558bf0565acf9840fa1c4f